pkgs/applications/video/obs-studio/wrapper.nix

```diff
@@ -6,19 +6,29 @@ symlinkJoin {
   name = "wrapped-${obs-studio.name}";
 
   nativeBuildInputs = [ makeWrapper ];
-  paths = [ obs-studio ] ++ plugins;
+  paths = [ obs-studio ];
```

Comment:
So, here... we need `obs-studio`'s bin, and `plugins`' bin.

Reply:
Does your test above cause something to break here? I can run that if so.

If we revert this line it puts the plugin .so and data back in the default directories which I think will re-introduce the double-load problem.

Reply:
I'm not sure how the plugins are getting paths to their bin files, but if they're relative to the .so locations my previous version might be an easier win.

Reply:
> If we revert this line it puts the plugin .so and data back in the default directories which I think will re-introduce the double-load problem.

OBS will load stuff from the original `obs-studio` derivation, as that is where its real executable sits, and then load plugins from your new `symlinkJoin`s thanks to the envvars.

I just noted you're right in removing the `++ plugins`... The problem here is a little more fundamental, `wrapOBS` is making a small FHS when it shouldn't. If we need something more from these plugins, we should add them to `systemPackages`.

Reply:
We'll break setups where people (like me) are using this "small FHS" for having access to binaries and libraries of the plugin, but stop it from going into a headache later.

Reply:
Ah ha, so it's for external usage. Got it. Yeah, in that case standalone plugin installation seems like the right answer. There's probably a better version of wrapOBS that would take a major refactor but I'm too green to know what it looks like.